hebiros_gazebo_plugin/plugin/hebiros_gazebo_plugin.cpp

```diff
@@ -14,12 +14,16 @@ void HebirosGazeboPlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf) {
   int argc = 0;
   char **argv = NULL;
   ros::init(argc, argv, "hebiros_gazebo_plugin_node");
-  this->n.reset(new ros::NodeHandle);
 
-  this->add_group_srv =
```

Comment:
Why did the `add_group` service get removed here?

Reply:
I mistakenly included that (from https://github.com/HebiRobotics/HEBI-ROS/pull/104/files)  when I copied over changes. I'll fix it.